src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({entries, updateLike}) => {
+    const chatData = entries.map((entry) => {
+        return (
+            <div key={entry.id}>
+                <ChatEntry
+                    id={entry.id}
+                    sender={entry.sender}
+                    body={entry.body}
+                    timeStamp={entry.timeStamp}
+                    liked={entry.liked}
+                    updateLike={updateLike}
+                />
+            </div>
+        );
+    });
+
+    return (
+        <div className="chat-log">
+            {chatData}
+        </div>
+    );
+};
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.number.isRequired,
+            sender: PropTypes.string.isRequired,
+            body: PropTypes.string.isRequired,
+            timeStamp: PropTypes.string.isRequired,
+            liked: PropTypes.bool.isRequired,
+        })
+    ), //.isRequired, 
```

Comment:
Make sure you mark the array itself as required! It's essential that the chat entries are received as a prop.